src/boot/boot_main.c

```diff
@@ -1,4 +1,3 @@
-#include <ultra64.h>
```

Comment:
I don't really agree with removing `ultra64.h` includes from files, since they will all have to be added back later when we split headers. I know some files haven't been consistent about including it tho, and it would be hard to enforce it right now, but I just want to make sure you're aware that this particular change is probably going to add more work for when we split headers.

Reply:
@Roman971 there was some discussion on this in discord, your input would be appreciated

Reply:
Well I don't really have more input on this besides what I said in this comment. I don't think we should be removing these includes even tho it's technically not "required" atm with our giant headers.